.github/dependabot.yml

```diff
@@ -42,55 +42,59 @@ updates:
   - package-ecosystem: "gomod"
     directory: "/src/go/docker-credential-none"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     open-pull-requests-limit: 1
     labels: ["component/dependencies"]
-    reviewers: [ "Nino-K" ]
+    reviewers: [ "ericpromislow" ]
 
   - package-ecosystem: "gomod"
     directory: "/src/go/extension-proxy"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     open-pull-requests-limit: 1
     labels: ["component/dependencies"]
     reviewers: [ "Nino-K" ]
 
   - package-ecosystem: "gomod"
     directory: "/src/go/mock-wsl"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     open-pull-requests-limit: 1
     labels: ["component/dependencies"]
     reviewers: [ "Nino-K" ]
 
   - package-ecosystem: "gomod"
     directory: "/src/go/nerdctl-stub"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     open-pull-requests-limit: 1
     labels: ["component/dependencies"]
     reviewers: [ "Nino-K" ]
 
   - package-ecosystem: "gomod"
     directory: "/src/go/rdctl"
+    ignore:
+      - dependency-name: "github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/generated/options"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     open-pull-requests-limit: 1
     labels: ["component/dependencies"]
-    reviewers: [ "Nino-K" ]
+    reviewers: [ "ericpromislow" ]
 
   - package-ecosystem: "gomod"
     directory: "/src/go/vtunnel"
     schedule:
-      interval: "weekly"
+      interval: "daily"
     open-pull-requests-limit: 1
     labels: ["component/dependencies"]
     reviewers: [ "Nino-K" ]
 
   - package-ecosystem: "gomod"
     directory: "/src/go/wsl-helper"
+    ignore:
+    - dependency-name: "github.com/rancher-sandbox/rancher-desktop/src/go/wsl-helper/pkg/dockerproxy/models"
```

Comment:
I would suggest to add a comment here so we know in the future why we decided to ignore it.

Reply:
done